datahub-web-react/build.gradle

```diff
@@ -5,10 +5,15 @@ node {
 
   // If true, it will download node using above parameters.
   // If false, it will try to use globally installed node.
-  download = true
+
+  if (project.hasProperty('useSystemNode') && project.getProperty('useSystemNode').toBoolean()) {
+    download = false
+  } else {
+    download = true
+  }
 
   // Version of node to use.
-  version = '14.15.3'
+  version = '16.8.0'
```

Comment:
@gabe-lyons is this node upgrade ok?

Reply:
yes! I've been running datahub using 16.4.2 for a while and it's been just fine